packages/paste-core/components/chat-log/src/ChatAttachmentDescription.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {BoxElementProps} from '@twilio-paste/box';
+
+import {Text} from '@twilio-paste/text';
+
+export interface ChatAttachmentDescriptionProps {
+  children: string;
+  element?: BoxElementProps['element'];
+}
+
+const ChatAttachmentDescription = React.forwardRef<HTMLElement, ChatAttachmentDescriptionProps>(
+  ({children, element = 'CHAT_ATTACHMENT_DESCRIPTION', ...props}, ref) => {
+    return (
+      <Text
+        element={element}
+        ref={ref}
+        as="div"
+        fontSize="fontSize20"
+        color="colorTextWeak"
+        lineHeight="lineHeight10"
+        {...props}
```

Comment:
We need to safelySpreadTextProps here. Also, do note that you're spreading after applying your styles. So folks can override fontSize / color / etc.  Is this intended?

Reply:
Ah, it is not! just updated it to use safelySpreadTextProps and changed the order. I also changed the order for props spreading in `ChatAttachment` and now use safeSpreadBoxProps in `ComposerAttachmentCard`